docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+﻿version: '3.8'
+
+services:
+  sea:
+    build: .
+    ports:
+      - "5001:80"
+    environment:
+      ASPNETCORE_ENVIRONMENT: "Development"
+    volumes:
+      # map the dotnet user-secret folder
+      - $APPDATA/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
+  db:
+    image: postgres
+    restart: unless-stopped
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_DB: seadev
+    ports:
+      - "5432:5432"
```

Comment:
It's probably fine as a default but this will need a note in the Z2H that if you're already running a local postgres instance they'll fight for this port.

Reply:
will add!